Jenkinsfile

```diff
@@ -76,7 +76,22 @@ pipeline {
                 sh 'cd config;. venv/bin/activate; kgx transform --input-type tsv --output-type nt -o ./merged-kg.nt merged-kg.tar.gz'
                 sh 'cd config;. venv/bin/activate; pigz merged-kg.nt'
             }
-        }        
+        }
+        stage('Make blazegraph journal'){
+            steps {
+                dir('./config/blazegraph') {
+                        git(
+                                url: 'https://github.com/balhoff/blazegraph-runner.git',
+                                branch: 'master'
+                        )
+                        sh 'sbt stage'
+                        sh 'cd ..'
+                        sh 'pigz -d ../merged-kg.nt.gz'
+                        sh './blazegraph-runner/target/universal/stage/bin/blazegraph-runner/blazegraph-runner load --informat=ntriples --journal=merged-kg.jnl --use-ontology-graph=true ./merged-kg.nt'
```

Comment:
@justaddcoffee I may be missing something but I think this path should just be `./blazegraph-runner/target/universal/stage/bin/blazegraph-runner`.

Reply:
Thanks @balhoff ! You are right